xcube/webapi/ows/coverages/controllers.py

```diff
@@ -0,0 +1,312 @@
+# The MIT License (MIT)
+# Copyright (c) 2023 by the xcube team and contributors
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+import os
+import tempfile
+from typing import Mapping, Sequence, Optional
+
+import numpy as np
+import pyproj
+import xarray as xr
+
+from xcube.webapi.datasets.context import DatasetsContext
+
+
+def get_coverage_as_json(ctx: DatasetsContext, collection_id: str):
+    """
+    Return a JSON representation of the specified coverage
+
+    Currently, the range set component is omitted.
+
+    :param ctx: a dataset context
+    :param collection_id: the ID of the collection providing the coverage
+    :return: a JSON representation of the coverage
+    """
+    return {
+        'id': collection_id,
+        'type': 'CoverageByDomainAndRange',
+        'envelope': get_collection_envelope(ctx, collection_id),
+        'domainSet': get_coverage_domainset(ctx, collection_id),
+        'rangeSet': {
+            'type': 'RangeSet',
+            'dataBlock': {'type': 'VDataBlock', 'values': ['TODO']},  # FIXME
```

Comment:
I think you should write more about why this is not added.

Reply:
I agree, and I've now done so.